### PR TITLE
Add daemonization.

### PR DESCRIPTION
**Pull request by gavofyork**

### CODE REVIEW

File: parity/main.rs [hunk 1]
```diff
@@ -171,6 +180,71 @@ impl Configuration {
...
 	}
+
+	fn execute(&self) {
+		if self.args.flag_version {
+			print_version();
+			return;
+		}
+		if self.args.cmd_daemon {
+			let daemonize = Daemonize::new()
+				.pid_file("/tmp/parity.pid")	// Every method except `new` and `start`
```
Comment (tomusdrw) [inline parity/main.rs]: I guess it won't work on Windows? :)

File: parity/main.rs [hunk 2]
```diff
@@ -55,12 +58,14 @@ Parity. Ethereum Client.
   Copyright 2015, 2016 Ethcore (UK) Limited
 
 Usage:
+  parity daemon [options] [ --no-bootstrap | <enode>... ]
```
Comment (tomusdrw) [inline parity/main.rs]: Why do we need it anyway? Why it's not left for user to deamonize?

**Comment by gavofyork** [comment]
not sure what daemonize does under windows...

File: parity/main.rs (hunk 2 above)
Comment (gavofyork) [inline parity/main.rs]: not sure what daemonize does under windows...

File: parity/main.rs [hunk 3]
```diff
@@ -55,12 +58,14 @@ Parity. Ethereum Client.
...
   parity [options] [ --no-bootstrap | <enode>... ]
 
 Options:
   --chain CHAIN            Specify the blockchain type. CHAIN may be either a JSON chain specification file
-                           or frontier, mainnet, morden, or testnet [default: frontier].
+						   or frontier, mainnet, morden, or testnet [default: frontier].
```
Comment (arkpar) [inline parity/main.rs]: misaligned?

**Comment by gavofyork** [comment]
yeah - still needs work.

**Comment by gavofyork** [comment]
merging since this compiles locally and build failure is unrelated.

**Pull request merged**